src/CustomButton.js

```diff
@@ -0,0 +1,14 @@
+import React, {
+  Component
+} from 'react';
+import './App.css';
+
+class CustomButton extends Component {
+  render() {
+    return ( <button { ...this.props}>{this.props.id}</button>        
```

Comment:
Please remove spaces between `(` and ` <` etc

Reply:
done